libs/cardano-ledger-pretty/cardano-ledger-pretty.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-pretty
-version:            1.3.1.0
+version:            1.3.2.0
```

`1.3.1.0` hasn't been released yet. You can learn this info from the CHANGELOG (latest version in it matches the current one), from git tag or from CHaP
```suggestion
version:            1.3.1.0
```